app/models/reply_validation.rb

```diff
@@ -0,0 +1,34 @@
+module ReplyValidation
+  extend ActiveSupport::Concern
+
+  attr_accessor :reply, :name, :email
+
+  included do
+    validates :reply, presence: { message: "Select yes if you want a reply" }
+    validates :email, email: { message: "Enter an email address in the correct format, like name@example.com" },
+                      if: :can_reply?,
+                      allow_blank: true
+    validate :validates_email_if_can_reply
+
+    validates :email, length: {
+      maximum: AppTicket::MAX_FIELD_CHARACTERS,
+      message: "The page field can be max #{AppTicket::MAX_FIELD_CHARACTERS} characters",
+    }
+    validates :name, length: {
+      maximum: AppTicket::MAX_FIELD_CHARACTERS,
+      message: "The page field can be max #{AppTicket::MAX_FIELD_CHARACTERS} characters",
```

Comment:
Change message to "Your name must be #{AppTicket::MAX_FIELD_CHARACTERS} characters or less"